.goreleaser.yaml

```diff
@@ -19,14 +19,8 @@ archives:
     name_template: >-
       {{ .ProjectName }}_
       {{- title .Os }}_
-      {{- if eq .Arch "amd64" }}x86_64
-      {{- else if eq .Arch "386" }}i386
-      {{- else }}{{ .Arch }}{{ end }}
-      {{- if .Arm }}v{{ .Arm }}{{ end }}
-    # use zip for windows archives
-    format_overrides:
-      - goos: windows
-        format: zip
+      {{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}
+
```

Comment:
so just remove the windows that would be enough?

Reply:
Since we planned the build for linux on two architecture which are`amd64 `and the other  is `arm64 `so the rest of those are not usable. Do we have any plans to generate the package for i386 as well?